### PR TITLE
Invalid board move

### PR DESCRIPTION
**Pull request by sarahabimay**
@maikon
@jsuchy

### CODE REVIEW

File: ttt/lib/ttt/board.ex [hunk 1]
```diff
@@ -1,16 +1,19 @@
 defmodule TTT.Board do
+  @x_mark "X"
```
Comment (felipesere) [inline ttt/lib/ttt/board.ex:2]: Not sold on how 'critical' these constants are. 
You use them in exactly one function.
Plus, id aim for `:x` and `:o`, as strings are purely for the UI.

File: ttt/lib/ttt/board.ex [hunk 2]
```diff
@@ -19,70 +22,64 @@ defmodule TTT.Board do
...
 
-  defp matrix_row_index(position, board) do
-    div(position, board_dimension(board))
+  defp found_winner_in_any_line?(lines) do
+     Enum.any?(lines, fn(line) -> all_same_mark?(line) end)
   end
 
-  defp matrix_column_index(position, board) do
-    rem(position, board_dimension(board))
+  defp find_winning_mark(lines) do
```
Comment (felipesere) [inline ttt/lib/ttt/board.ex:44]: This can be

```
lines
|> Enum.any?(...)
|> Line.first
```

File: ttt/lib/ttt/board.ex (hunk 1 above)
Comment (sarahabimay) [inline ttt/lib/ttt/board.ex:2]: I have this down to hightlight.  Sloppy but I'm aware of it.  I have been rushing to get finished.

File: ttt/lib/ttt/board.ex (hunk 2 above)
Comment (sarahabimay) [inline ttt/lib/ttt/board.ex:44]: that's a line of Makis'!

**Pull request merged**